setup.py

```diff
@@ -499,6 +513,8 @@ def bundle_libzmq_extension(self):
             # include defines from zeromq msvc project:
             libzmq.define_macros.append(('FD_SETSIZE', 1024))
             libzmq.define_macros.append(('DLL_EXPORT', 1))
+            libzmq.define_macros.append(('_CRT_SECURE_NO_WARNINGS', 1))
+            libzmq.define_macros.append(('ZMQ_USE_SELECT', 1))
```

Comment:
When I don't set this, I get the following error:

    c:\users\acaron\desktop\develop\pyzmq\bundled\zeromq\src\poller.hpp(42) : fatal error C1189: #error :  None of the ZMQ_USE_* macros defined

I get this when building from master with out my changes too.  The one thing I'm doing differently is building against the latest ZeroMQ master, but I can't understand why that's creating the error.  Any ideas?